pkg/logging/json_logger.go

```diff
@@ -129,7 +130,15 @@ func (jsonLogger *JSONLogger) PostProcess() error {
 func viewLogsJSONFile(lvopts LogViewOptions, stdout, stderr io.Writer, stopChannel chan os.Signal) error {
```

Comment:
Is this function called in docker tests?

Reply:
Indirectly, yes

Reply:
Do you mean "is it called with target=docker"?

Reply:
Right, probably this should have another GitHub issue and PR?

Reply:
Oh, I see what you mean. Yes, I guess I can split it out and submit a separate PR.

Reply:
Pushed again without the above (will submit as separate PR).